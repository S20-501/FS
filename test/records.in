init -b 100 -s 1
enter -l 1 "file1.txt"
enter -l 1 "file2.txt"
enter -l 1 "file3.txt"
enter -l 1 "file4.txt"
enter -l 1 "file5.txt"
enter -l 1 "file6.txt"
enter -l 1 "file7.txt"
enter -l 1 "file8.txt"
enter -l 1 "file9.txt"
enter -l 1 "file10.txt"
enter -l 1 "file11.txt"
enter -l 1 "file12.txt"
enter -l 1 "file13.txt"
enter -l 1 "file14.txt"
enter -l 1 "file15.txt"
enter -l 1 "file16.txt"
enter -l 1 "file17.txt"
enter -l 1 "file18.txt"
enter -l 1 "file19.txt"
enter -l 1 "file20.txt"
enter -l 1 "file21.txt"
enter -l 1 "file22.txt"
enter -l 1 "file23.txt"
enter -l 1 "file24.txt"
enter -l 1 "file25.txt"
enter -l 1 "file26.txt"
enter -l 1 "file27.txt"
enter -l 1 "file28.txt"
enter -l 1 "file29.txt"
enter -l 1 "file30.txt"
enter -l 1 "file31.txt"
enter -l 1 "file32.txt"
enter -l 1 "file33.txt"
enter -l 1 "file34.txt"
enter -l 1 "file35.txt"
enter -l 1 "file36.txt"
enter -l 1 "file37.txt"
enter -l 1 "file38.txt"
enter -l 1 "file39.txt"
enter -l 1 "file40.txt"
enter -l 1 "file41.txt"
enter -l 1 "file42.txt"
enter -l 1 "file43.txt"
enter -l 1 "file44.txt"
enter -l 1 "file45.txt"
enter -l 1 "file46.txt"
enter -l 1 "file47.txt"
enter -l 1 "file48.txt"
enter -l 1 "file49.txt"
enter -l 1 "file50.txt"
enter -l 1 "file51.txt"
enter -l 1 "file52.txt"
enter -l 1 "file53.txt"
enter -l 1 "file54.txt"
enter -l 1 "file55.txt"
enter -l 1 "file56.txt"
enter -l 1 "file57.txt"
enter -l 1 "file58.txt"
enter -l 1 "file59.txt"
enter -l 1 "file60.txt"
enter -l 1 "file61.txt"
enter -l 1 "file62.txt"
enter -l 1 "file63.txt"
enter -l 1 "file64.txt"
exit
