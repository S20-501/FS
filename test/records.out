Filesystem created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
File created successfully.
Error in creating file: The available storage space has been exceeded.
change da world
my final message. Goodb ye.
